src/Npgsql/Replication/PgOutput/PgOutputStreamingMode.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Npgsql.Replication.PgOutput;
+
+/// <summary>
+/// Option to enable streaming of in-progress transactions.
+/// Minimum protocol version 2 is required to turn it on. Minimum protocol version 4 is required for the "parallel" option.
+/// </summary>
+public enum PgOutputStreamingMode
+{
+    /// <summary>
+    /// Disable streaming of in-progress transactions
+    /// </summary>
+    Off,
+
+    /// <summary>
+    /// Enable streaming of in-progress transactions
+    /// </summary>
+    On,
+
+    /// <summary>
+    /// Enable streaming of in-progress transactions and enable sending extra information with some messages to be used for parallelisation
+    /// </summary>
+    Parallel
```

Comment:
At least currently, conceptually this may be a bit of a "flags" enum... In other words, if PG introduces more options in addition to Parallel, we'd be in a bit of a bind as each bool feature would be independent and we'd have some explosion.

But I'm probably overthinking it.

Reply:
I doubt that there will ever be an additional option and at least to the backend it's not a flags enum but rather char constants: https://github.com/postgres/postgres/blob/b48f275f18d7da4f4863888ad047cbd699698880/src/include/catalog/pg_subscription.h#L175